<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Enter Command or Help:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_enter">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>enter</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_exit">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>exit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_quit">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>quit, no save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListWidget" name="listWidget_Stack"/>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_Registers"/>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_History"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget_Output"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>902</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menusigfig">
    <property name="title">
     <string>sigfig</string>
    </property>
    <addaction name="action_1"/>
    <addaction name="action0"/>
    <addaction name="action2"/>
    <addaction name="action4"/>
    <addaction name="action6"/>
    <addaction name="action8"/>
   </widget>
   <widget class="QMenu" name="menuOutputState">
    <property name="title">
     <string>OutputState</string>
    </property>
    <addaction name="actionfixed"/>
    <addaction name="actionfloat"/>
    <addaction name="actiongen"/>
   </widget>
   <widget class="QMenu" name="menuClear_Output">
    <property name="title">
     <string>Clear Output Area</string>
    </property>
    <addaction name="actionClear_Output"/>
   </widget>
   <addaction name="menusigfig"/>
   <addaction name="menuOutputState"/>
   <addaction name="menuClear_Output"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_1">
   <property name="text">
    <string>-1</string>
   </property>
  </action>
  <action name="action0">
   <property name="text">
    <string>0</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action4">
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="action6">
   <property name="text">
    <string>6</string>
   </property>
  </action>
  <action name="action8">
   <property name="text">
    <string>8</string>
   </property>
  </action>
  <action name="actionfixed">
   <property name="text">
    <string>fixed</string>
   </property>
  </action>
  <action name="actionfloat">
   <property name="text">
    <string>float</string>
   </property>
  </action>
  <action name="actiongen">
   <property name="text">
    <string>gen</string>
   </property>
  </action>
  <action name="actionClear_Output">
   <property name="text">
    <string>Clear Output</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
